src/tests/ftest/daos_perf/daos_perf.py

```diff
@@ -30,6 +32,8 @@ def test_small(self):
             daos_perf using single value type for 'LARGE' and 'R2s' object
             class. Run this config with multiple server/client configuration.
 
+        :avocado: tags=all,full_regression
+        :avocado: tags=hw,medium,ib2
```

Comment:
Is this truly a 'hw' test?  The `server_config` section of the yaml file does not require NVMe.

Reply:
Due to performance?

Reply:
Performance of the test or CI?  Typically given the limited availability of HW clusters we run any test that does not have a hard requirement for the HW clusters on the VM clusters.  In https://build.hpdd.intel.com/blue/organizations/jenkins/daos-stack%2Fdaos/detail/weekly-testing/179/pipeline/56 the runtimes for the various stages were:
- VM CentOS/Leap: 2h 11m 54s / 2h 18m 56s
- HW Small: 31m 37s
- HW Medium: 3h 3m 58s
- HW Large: 12h 29m 34s

Based on these numbers, for CI performance it make sense to run them on the VM clusters rather than HW Medium or Large - assuming the test takes less than ~50 minutes.

Reply:
Performance of the test, I believe the idea is to get performance numbers and avoid performance regressions. How long the test take to run? We don't know, the daos_perf tests are currently broken.

Reply:
Ok, then if this test is going to be used on the HW Medium cluster the yaml file will need some edits.  At most it can specify 4 hosts across the `test_servers` and `test_clients` lists - currently it specifies 8.  Two hosts can be used to launch 4 servers with something like https://github.com/daos-stack/daos/blob/master/src/tests/ftest/io/fio_small.yaml#L11-L33.  I suspect, however, that it would be simpler to just change this test to `hw,large`.

Reply:
hw large it is

Reply:
Fixed